src/main/kotlin/it/pagopa/wallet/services/WalletPaymentCDCEventDispatcherService.kt

```diff
@@ -25,13 +23,6 @@ class WalletPaymentCDCEventDispatcherService(
     private val walletExpireTimeout = Duration.ofSeconds(cdcQueueConfig.timeoutWalletExpired)
 
     fun dispatchEvent(event: BsonDocument): Mono<BsonDocument> =
```

Comment:
Would it be a good idea making this method return Mono<Response<SendMessageResult>> in order to enable the caller to implement some retry mechanism based on this result?

Reply:
The retry should be attempted up to N times? Maybe 3? 
What is the current approach in this cases?

Reply:
Retry mechanism added here 0f308dccdc885258f4a7b950c3acf8785da5081f
at WalletQueueClient.sendWalletEvent level